compiler/suggest.nim

```diff
@@ -109,7 +109,11 @@ proc symToSuggest(conf: ConfigRef; s: PSym, isLocal: bool, section: IdeCmd, info
         result.qualifiedPath.add(ow2.origModuleName)
       if ow != nil:
         result.qualifiedPath.add(ow.origModuleName)
-    result.qualifiedPath.add(s.name.s)
+    if s.name.s[0] in OpChars + {'[', '{', '('} or
+       s.name.s in specialWords[wAddr..wYield]:
```

Comment:
There is a more efficient way to implement this check like  ``s.name.id in ord(wAddr)..ord(wYield)``

Reply:
Done :)